src/resmom/mom_hook_func.c

```diff
@@ -3099,45 +3547,37 @@ record_job_last_hook_executed(unsigned int hook_event,
  * @return	int
  * @retval 	0 means at least one hook was encountered to have rejected the
  *		request.
- * @retval  	1 means all the executed hooks have agreed to accept the request
+ * @retval 	1 means all the executed hooks have agreed to accept the request
  * @retval	2 means no hook script executed (special case).
- * @retval      -1 an internal error occurred
+ * @retval	-1 an internal error occurred
+ * @retval	HOOK_RUNNING_IN_BACKGROUND 
+ * 				background process started for the hook script.
  *
  */
 int
 mom_process_hooks(unsigned int hook_event, char *req_user, char *req_host,
 	mom_hook_input_t *hook_input, mom_hook_output_t *hook_output, char *hook_msg,
 	size_t msg_len, int update_server)
 {
-	hook			*phook;
-	hook			*phook_next = NULL;
+	char		hook_infile[MAXPATHLEN+1];
+	char		hook_outfile[MAXPATHLEN+1];
+	char		hook_datafile[MAXPATHLEN+1];
+	char		*log_id = NULL;
 	int			rc;
-	pbs_list_head		*head_ptr;
 	int			num_run = 0;
-	char			hook_infile[MAXPATHLEN+1];
-	char			hook_outfile[MAXPATHLEN+1];
-	char			hook_datafile[MAXPATHLEN+1];
-	int			accept_flag = 1;
-	int			reject_flag = 0;
-	char			reject_msg[HOOK_MSG_SIZE+1];
-	int			reject_rerunjob = 0;
-	int			reject_deletejob = 0;
-	int			reboot_flag = 0;
-	char			reboot_cmd[HOOK_BUF_SIZE];
-	int			wait_hook;
-	pbs_list_head		vnl_changes;
 	int			set_job_exit = 0;
-	char			*log_id = NULL;
 	int			log_type = 0;
 	int			log_class = 0;
-	job			*pjob = NULL;
-	hook			**last_phook = NULL;
-	/* last hook that executed */
-	unsigned int		*pfail_action = NULL; 	/* holds summation of hook */
-	/* fail_action values */
-	/* of those hook that */
-	/*  executed */
 	int			*reject_errcode = NULL;
+	unsigned int		*pfail_action = NULL;
+	hook		*phook;
+	hook		*phook_next = NULL;
+	hook		**last_phook = NULL;
+	job			*pjob = NULL;
+	pbs_list_head		vnl_changes;
+	pbs_list_head		*head_ptr;
+	mom_process_hooks_params_t		*php = NULL;
+	struct work_task task;
```

Comment:
the variable names need not be aligned, and in the above code, they are neither aligned nor placed after the type.